cms/djangoapps/contentstore/views/public.py

```diff
@@ -7,12 +7,17 @@
 from django.shortcuts import redirect
 from urllib.parse import quote_plus  # lint-amnesty, pylint: disable=wrong-import-order
 from waffle.decorators import waffle_switch
+from django.contrib import admin
 
 from common.djangoapps.edxmako.shortcuts import render_to_response
+from openedx.core.djangoapps.user_authn.config.waffle import REDIRECT_TO_LMS
 
 from ..config import waffle
 
-__all__ = ['register_redirect_to_lms', 'login_redirect_to_lms', 'howitworks', 'accessibility']
+__all__ = [
+    'register_redirect_to_lms', 'login_redirect_to_lms', 'howitworks', 'accessibility',
+    'redirect_to_lms_login_for_admin'
```

Comment:
```suggestion
    'redirect_to_lms_login_for_admin',
```
[nit] Best to have a trailing comma in multiline lists like this so that future diffs are smaller.